metagraph/src/graph/representation/succinct/dbg_succinct.hpp

```diff
@@ -120,6 +120,10 @@ class DBGSuccinct : public DeBruijnGraph {
     virtual bool load_without_mask(const std::string &filename_base) final;
     virtual bool load(const std::string &filename_base) override;
     virtual void serialize(const std::string &filename_base) const override;
+    // initialize DBGSuccinct and dump to disk without loading everything in RAM
+    static void serialize(boss::BOSS::Chunk&& chunk,
```

Comment:
This interface is a little dangerous because it allows for incomplete BOSS chunks to be serialized with the same file extension as full graphs.

We should at least have a warning here so people don't use it inappropriately.

Reply:
No-no, this thing does initialize a DBGSuccinct and serializes that, not the input graph chunk.